ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/winc-fips/cucushift-installer-rehearse-aws-ipi-ovn-winc-fips-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-winc-fips
+  steps:
+    env:
+      SUB_CHANNEL: stable
+      SUB_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
+      SUB_PACKAGE: windows-machine-config-operator
+      SUB_TARGET_NAMESPACES: '!install'
+      SUB_SOURCE: "qe-app-registry"
+      WINDOWS_NODE_REPLICAS: 2
+      FIPS_ENABLED: "true"
```

Comment:
Generally almost all the existing install workflows are using `ipi-conf`, you can easily set `FIPS_ENABLED: "true"` when you are adding a ci job, not have to introduce a dedicated FIPS install workflow.

Another question, here the `fips` testing is for covering windows nodes ? If yes, I have concerns that `FIPS_ENABLED: "true"` will enable FIPS on windows nodes. As far as I know, the option is respected by rhcos node, while the windows nodes are created by your team's script, I am not sure if the option are also respected by windows nodes, and enable FIPS on the system level. I have no knowledge about how to validate FIPS enabled or not in windows, need future investigation on your own.